models/task/task.go

```diff
@@ -23,11 +23,13 @@ type TaskStruct struct {
 	IssueNumber sql.NullInt64
 	Title       string
 	Description string
+	PullRequest bool
+	HtmlURL     sql.NullString
 	database    db.DB
 }
 
-func NewTask(id int64, listID int64, projectID int64, userID int64, issueNumber sql.NullInt64, title string, description string) *TaskStruct {
-	task := &TaskStruct{Id: id, ListId: listID, ProjectId: projectID, UserId: userID, IssueNumber: issueNumber, Title: title, Description: description}
+func NewTask(id int64, listID int64, projectID int64, userID int64, issueNumber sql.NullInt64, title string, description string, pullRequest bool, htmlURL sql.NullString) *TaskStruct {
```

Comment:
__[golint]__
* [golint] exported function NewTask should have comment or be unexported